### ISSUE TITLE
val town

### ISSUE THREAD

**Issue by hfu** [UNopenGIS/7#365]
This looks like fun! 
https://www.val.town/v/tmcw/poll

**Comment by hfu** [comment]
@yuiseki val town の強烈な思想は https://blog.val.town/blog/end-programmer-programming/ に書かれていますw。
#363

**Comment by yuiseki** [comment]
ついムキになって地図を表示してしまいました。

https://www.val.town/v/yuiseki/untitled_amaranthSnake
https://yuiseki-untitled_amaranthsnake.web.val.run/

**Comment by hfu** [comment]
@yuiseki My fork of your val is https://www.val.town/v/hfu/react_maplibre. Thank you for sharing! Without your input I could not implement it. 

@stevekrouse Excited to see you here! We are a group of geospatial experts who would like to contribute web map technology for UN (peacekeeping) operations. Being geospatial expert, we admire @tmcw for his products and that is why I found Val Town. 

We believe we can keep web maps open by sharing cloud-native geospatial data and make use of that data by web-based notebook solutions. I am excited to know about Val Town because as I understand Val Town can store states or small geospatial data of "temporal concern". 

We also like https://blog.val.town/blog/end-programmer-programming/ because many of us think in a similar way in our community :-) 

@yuiseki and all, I appreciate your input about your expectation or other comments on Val Town!

**Comment by yuiseki** [comment]
Great to see you @stevekrouse !

I don't think further explanation is necessary as @hfu has explained the objectives of our group beautifully.

So, I want to talk about why I personally got hooked on val.town.
I've only heard briefly about val.town from @hfu, but I strongly agree with the idea that all humans should be able to do programming.
So I signed up for val.town.
The sign-up and onboarding process at val.town was quite brilliant and strongly inspiring me.
(I am the Product Manager of Gyazo, the one of world's largest screenshot and screen record cloud sharing application. This is why I have such a strong interest in these details of the product.)

Well, enough of my personal story.

I would like to share my thoughts with our group about the possibilities of utilizing val.town.
We are a group of geospatial information experts, but we are unique in that we place the highest priority on (free and) open source software and open data.
Because, we, the UN Smart Maps Group, are the seventh working group of the UN **Open GIS** Initiative.
And the UN Open GIS Initiative's primary mission is to make the UN's (peacekeeping operations) geospatial information system to replaceable with (free and) open source software and open data.
This is why I used OpenStreetMap data in val.town to display the map, but there is more to follow.
OpenStreetMap has an important policy of not dealing with "fluid" geospatial information.
Fluid geospatial information means geospatial information related to incidents, accidents, disasters, conflicts, etc.
Because the objective of OpenStreetMap is to provide reliable and stable maps that are properly maintained.
Nevertheless, as I worked with this group, I gained insight into the importance of being able to properly handle such information with (free and) open source software and open data, which I named "temporary concerns" or "temporal concern".
In all probability, no UN geospatial information system would be complete without such information.
(Note: the term "Concern" is inspired from Ruby on Rails: https://www.writesoftwarewell.com/how-rails-concerns-work-and-how-to-use-them/ )
I have just started exploring val.town, however, as @hfu says, val.town may be useful in dealing with such (geospatial) temporary concerns or temporal concerns.

Lastly, let me to write another personal story,
My ambition is to maximize the desirable possibilities of humanity.
For this reason, recently days, I am strongly focused on generative AI, starting with LLMs, and find it interesting that val.town has also raised the linkage with the OpenAI API as an example.

Sorry for the long post. Thank you for reading to the end!
